docs/pages/admin-guides/infrastructure-as-code/terraform-provider/ci-or-cloud.mdx

```diff
@@ -0,0 +1,394 @@
+---
+title: Run the Teleport Terraform Provider in CI or Cloud
+description: How to manage dynamic resources using the Teleport Terraform provider from your CI pipelines or Cloud provider.
+---
+
```

Comment:
I would add a brief intro paragraph here that explains the scope of this guide so a user knows whether to keep reading.